test/unit/review.test.js

```diff
@@ -0,0 +1,28 @@
+const { generateReviewData } = require('../../data-gen/review');
+
+describe('Review data generation', () => {
+  test('generates an array of 1000 review objects with correct data fields', () => {
+    console.log = jest.fn();
+    let reviewData = generateReviewData();
+    expect(console.log).toHaveBeenCalled();
+    expect(reviewData.length).toBe(1000);
+    reviewData.forEach(reviewObj => {
+      expect(reviewObj).toMatchObject({
```

Comment:
A potential additional test this could be to check if all 100 games have 10 reviews each, this concept could be applied to your `user.test.js` for uniqueness/ randomness

Reply:
Noted, making this change now for more thorough tests. Thanks!